frontend/src/components/app/app.tsx

```diff
@@ -42,14 +41,6 @@ const App: FC = () => {
             </AuthorizedWrapper>
           }
         />
-        <Route
-          path={AppRoute.ANY}
-          element={
-            <AuthorizedWrapper>
-              <NotFound />
-            </AuthorizedWrapper>
-          }
-        />
```

Comment:
Why it was removed?

Reply:
Already fixed)

Reply:
@WilhelmWesser can you push the commit to revert NotFound component deletion?

Reply:
@v0ldemar01 Strange thing happend, did not notice how deleted it. Now, I am sure it is fixed